app/helpers/users_helper.rb

```diff
@@ -9,4 +9,19 @@ def sortable_full_name(user)
     [user.last_name, user.first_name].reject { |n| n.nil? or n.blank? }.join(', ')
   end
 
+  def full_name_list(users)
+    list = []
+    users.each do |user|
+      list << full_name(user)
+    end
+    list.to_sentence
```

Comment:
this (or some version of it) also works:
`users.map{ |user| full_name(user) }.to_sentence`